libraries/botbuilder-dialogs/src/skillDialog.ts

```diff
@@ -72,10 +71,9 @@ export class SkillDialog extends Dialog {
 
         // Store delivery mode and connection name in dialog state for later use.
         dc.activeDialog.state[this.DeliveryModeStateKey] = dialogArgs.activity.deliveryMode;
-        dc.activeDialog.state[this.SsoConnectionNameKey] = dialogArgs.connectionName;
 
         // Send the activity to the skill.
-        const eocActivity = await this.sendToSkill(dc.context, skillActivity, dialogArgs.connectionName);
+        const eocActivity = await this.sendToSkill(dc.context, skillActivity, this.dialogOptions.connectionName);
```

Comment:
This bit can be cleaned up: `sendToSkill()` simply forwards the connectionName to `interceptOAuthCards()`.

Since `interceptOAuthCards()` is bound to the instance, we don't need to pass the connectionName as a parameter.